qt/tools/genhooks_gui.py

```diff
@@ -58,6 +58,27 @@ def on_overview_will_render_content(overview, content):
         legacy_hook="showAnswer",
         legacy_no_args=True,
     ),
+    Hook(
+        name="reviewer_will_init_answer_buttons",
+        args=[
+            "buttons_tuple: Tuple[Tuple[int, str], ...]",
+            "reviewer: aqt.reviewer.Reviewer",
+            "card: Card",
+        ],
+        return_type="Tuple[Tuple[int, str], ...]",
+        doc="""Used to modify list of answer buttons
+
+        buttons_tuple is a sequence of buttons, with each button represented
+        by a tuple containing an int for the button's number and a string for
```

Comment:
Super minor, but I would suggest calling this value `ease' for consistency.

Reply:
> I would suggest calling this value `ease' for consistency.

Oh, fair point. How about ```ease_buttons_tuple``` though?

I just worry `ease` gets a little overloaded -- it's 

- the multiplier for the interval (usually called 'factor' in the code), 
- the numeric answer (1-4), 
- here I'm talking about the actual buttons with labels that point to that second meaning, so about half a step removed?

Reply:
Hmm, took a look at renaming, we have _answerButtonList() in the existing code before this, realized that's what I was matching. 

Changing this to `will_init_ease_buttons` or `ease_buttons_tuple` starts pulling on a thread... how far do we want to go? Refactor `_answerButtonList` to `_easeButtonList`?

Might be safer to leave it as submitted, I don't know.

Reply:
Hmm, sorry, I think my comment might have been a bit too vague. I was only thinking about changing the naming in the docstring from "number" to "ease", since other hooks like `reviewer_will_answer_card` also use `ease` to describe the button index.

You're definitely right in that `ease` is a bit ambiguous in Anki's code base. I guess both terms have their merits here: We could go with `ease` and be ambigiously consistent with other hooks and Anki's variable naming, or go with `button number` or something similar and be more clear, but less consistent.

@dae, super minor point, so don't want to bother you with this too much, but any thoughts?

Reply:
Oh, yeah, docstring is the easy fix, definitely agree, updated.

Reply:
The new graphs code and Rust code are using 'button' in preference to 'ease'
https://github.com/ankitects/anki/blob/master/rslib/src/revlog.rs#L20